benchmarks/transport/CMakeLists.txt

```diff
@@ -33,3 +33,23 @@ if (GHEX_USE_UCP)
         endforeach()
     endif()
 endif()
+
+if (GHEX_BUILD_FORTRAN)
+    enable_language(Fortran)
```

Comment:
is the `enable_language` needed (since it is already done in the main CMakeLists.txt)?